.github/check_agda.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+export Agda_datadir=$PWD/data
+export PATH=$PWD/bin:$PATH
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  # See https://gitlab.haskell.org/ghc/ghc/-/issues/17418
+  xattr -rc $PWD/bin
+fi
+
+echo "The nightly version of Agda is"
+agda --version
+
+echo "Checking if Agda nightly is runnable on your system..."
+for prim in ./data/lib/prim/Agda/**/*.agda
```

Comment:
This `**`  pattern requires `globstar`, which is present in Bash >= 4. However, macOS ships with an ancient 3.2 version. On the macOS runners it's likely only matching the `.agda` under that first directory, rather than recursively. So, I think this needs to use `find`.

Reply:
Done. I wasn't aware of this until I realise my bash has been updated to 5 ...